parse_test.go

```diff
@@ -20,7 +20,17 @@ type gppTestData struct {
 
 func TestParse(t *testing.T) {
 	testData := map[string]gppTestData{
-		"GPP-tcf": {
+		"gpp-tcf": {
+			description: "GPP string with EU TCF V2",
+			gppString:   "DBABM~CPXxRfAPXxRfAAfKABENB-CgAAAAAAAAAAYgAAAAAAAA",
```

Comment:
Updated examples to exactly match GPP docs. Somewhere along the way we added extra padding to those examples, which cause us to overlook this bug.